### ISSUE TITLE
Tagging with images

### ISSUE THREAD

**Issue by eliluong** [tagspaces/tagspaces#344]
Is it possible to be able to add an image/cover to a file to use as a tag, or is this not the intended purpose of tagspaces?

**Comment by uggrock** [comment]
How do you mean it "tagging with images", can you explain a little more? 

Starting with version 2.0.3 you can set a cover image/thumbnail for every file just by placing a file with the  same name +".png" in a  ".ts" folder. So for example for /photos/coolimage.jpg you have to create /photos/.ts/coolimage.jpg.png .

**Comment by eliluong** [comment]
I have multiple video clips in one directory and would like to have an image representation of each file. So I updated to 2.0.3 (running under windows, x64). I made a .ts folder from command line. I have a file video.mkv and in the .ts folder I put video.jpg.png. Then I set up a new location in Tagspaces, but nothing is showing up in Grid view or List view.

I guess tagging with images is not the correct phrase I meant to use, but instead to be able to set a cover image like you said.
